src/SkipLinks.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { forwardRef, memo } from "react";
+import { assert, Equals } from "tsafe";
+import { symToStr } from "tsafe/symToStr";
+import { fr } from "./lib";
+import { createComponentI18nApi } from "./lib/i18n";
+import { cx } from "./lib/tools/cx";
+
+export type SkipLink = {
+    label: string;
+    anchor: string;
+};
+
+export type SkipLinksProps = {
+    className?: string;
+    links: SkipLink[];
+    classes?: Partial<Record<"root" | "list" | "link", string>>;
+};
+
+/** @see <https://react-dsfr-components.etalab.studio/?path=/docs/components-skiplinks> */
+export const SkipLinks = memo(
+    forwardRef<HTMLDivElement, SkipLinksProps>((props, ref) => {
+        const { className, classes = {}, links, ...rest } = props;
+        const { t } = useTranslation();
+        assert<Equals<keyof typeof rest, never>>();
+
+        return (
+            <div className={cx(fr.cx("fr-skiplinks"), classes.root, className)} ref={ref} {...rest}>
+                <nav className={fr.cx("fr-container")} role="navigation" aria-label={t("label")}>
+                    <ul className={cx(fr.cx("fr-skiplinks__list"), classes.list)}>
+                        {links &&
+                            links.map(link => (
+                                <li key={link.anchor}>
+                                    <a
+                                        className={cx(fr.cx("fr-link"), classes.link)}
+                                        href={`#${link.anchor}`}
```

Comment:
```suggestion
                                        href={link.anchor}
```
Pour être choérent avec l'utilisation que tu met en avant (en l'état ça fait un double hashtag)

Reply:
Je voulais rester générique pour permettre tous types de liens, mais autant restreindre, normalement les liens d'évitements sont sensé se faire sur la page courante.